app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Constraints.scala

```diff
@@ -121,4 +121,11 @@ trait Constraints {
       Invalid(invalidKey, length)
     case _                               => Valid
   }
+
+  protected def areAllElementsNumbers(invalidKey: String): Constraint[String] = Constraint {
+    case input if !(input forall Character.isDigit) =>
+      Invalid(invalidKey)
+    case _                                          => Valid
+
+  }
```

Comment:
```suggestion
  protected def onlyContainsNumbers(invalidKey: String): Constraint[String] = Constraint {
    case input if input.exists(!_.isDigit) =>
      Invalid(invalidKey)
    case _                                          => Valid
  }
```

This is also missing a unit test in `ConstraintsSpec`

Reply:
Done & Done :)